### PR TITLE
BETA: Register imsoumyaneel.is-a.dev

### PR DESCRIPTION
**Pull request by A-Little-Hat**
Added `imsoumyaneel.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/imsoumyaneel.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "A-Little-Hat",
+        "email": "soumyasarkar309@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://test-3e8b2.web.app/"
```
Comment (Stef-00012) [inline domains/imsoumyaneel.json:7]: Invalid CNAME record (can not contain `https://` or `/`)

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Review by Stef-DP — APPROVED** [review]
Seems good now

File: domains/imsoumyaneel.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "A-Little-Hat",
+        "email": "soumyasarkar309@gmail.com"
+    },
+    "record": {
+        "CNAME": "test-3e8b2.web.app/"
```
Comment (Stef-00012) [inline domains/imsoumyaneel.json:7]: nevermind i just rechecked and your CNAME record is invalid, can not contain `/`

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Comment by A-Little-Hat** [comment]
I updated the repo. Please review it.

**Comment by Stef-00012** [comment]
Now seems good

**Review by iostpa — APPROVED** [review]

**Pull request merged**